src/scripts/main.js

```diff
@@ -21,7 +21,39 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  document.body.appendChild(ul);
```

Comment:
The tree should be appended to the `element` parameter, not directly to `document.body`. This ensures the function uses its arguments correctly.